config/webpack.config.dev.js

```diff
@@ -171,7 +171,7 @@ module.exports = {
       inject: true,
       template: paths.appHtml,
       cspPlugin: { //adds CSP Meta tag plugin
-        enabled: true,
+        enabled: false,
```

Comment:
local testing didn't work with CSP enabled, as it rejects all requests to non-production domains